src/Uno.UI/UI/Xaml/Controls/MenuFlyout/MenuFlyoutPresenter.cs

```diff
@@ -1,13 +1,958 @@
+		}
+
+		void ResetVisualState()
+		{
+			VisualStateManager.GoToState(this, "None", false);
+		}
+
+		private protected override void ChangeVisualState(
+		   // true to use transitions when updating the visual state, false
+		   // to snap directly to the new visual state.
+		   bool bUseTransitions)
+		{
+		}
+
+		protected override void OnApplyTemplate()
+		{
+			base.OnApplyTemplate();
+
+			ScrollViewer spScrollViewer;
+
+			// Get the ScrollViewer template part
+			spScrollViewer = this.GetTemplateChild("MenuFlyoutPresenterScrollViewer") as ScrollViewer;
+			m_tpScrollViewer = spScrollViewer;
+
+			// Apply a shadow
+
+			// UNO TODO
+			//bool isDefaultShadowEnabled = IsDefaultShadowEnabled;
+			//if (isDefaultShadowEnabled)
+			//{
+			//	DependencyObject spChild;
+			//	VisualTreeHelper.GetChildStatic(this, 0, &spChild);
+			//	var spChildAsUIE = spChild as UIElement;
+
+			//	if (spChildAsUIE != null)
+			//	{
+			//		ApplyElevationEffect(spChildAsUIE, GetDepth());
+			//	}
+			//}
+		}
+
+		// UNO TODO
+		//
+		//Timeline AttachEntranceAnimationCompleted(
+		//	string pszStateName,
+		//	uint nStateNameLength,
+		//	TimelineCompletedEventCallback pCompletedEvent)
+		//{
+		//	bool found = false;
+		//	VisualState spState;
+
+		//	VisualStateManager.TryGetState(this, pszStateName, null, &spState, &found);
+
+		//	if (found && spState)
+		//	{
+		//		Storyboard spStoryboard;
+
+		//		spStoryboard = (spState.Storyboard);
+		//		if (spStoryboard != null)
+		//		{
+		//			Timeline spTimeline;
+		//			(spStoryboard.As<ITimeline>(&spTimeline));
+
+		//			(pCompletedEvent.AttachEventHandler(spTimeline,
+		//				std.bind(&OnEntranceAnimationCompleted, this, _1, _2)));
+
+		//			(spTimeline.CopyTo(ppTimeline));
+		//		}
+		//	}
+		//}
+
+		void DetachEntranceAnimationCompletedHandlers()
+		{
+			// Marked as no longer used
+			//
+			//if (m_epTopPortraitCompletedHandler)
+			//{
+			//	ASSERT(m_tpTopPortraitTimeline);
+			//	(m_epTopPortraitCompletedHandler.DetachEventHandler(m_tpTopPortraitTimeline));
+			//	m_tpTopPortraitTimeline.Clear();
+			//}
+
+			//if (m_epBottomPortraitCompletedHandler)
+			//{
+			//	ASSERT(m_tpBottomPortraitTimeline);
+			//	(m_epBottomPortraitCompletedHandler.DetachEventHandler(m_tpBottomPortraitTimeline));
+			//	m_tpBottomPortraitTimeline.Clear();
+			//}
+
+			//if (m_epLeftLandscapeCompletedHandler)
+			//{
+			//	ASSERT(m_tpLeftLandscapeTimeline);
+			//	(m_epLeftLandscapeCompletedHandler.DetachEventHandler(m_tpLeftLandscapeTimeline));
+			//	m_tpLeftLandscapeTimeline.Clear();
+			//}
+
+			//if (m_epRightLandscapeCompletedHandler)
+			//{
+			//	ASSERT(m_tpRightLandscapeTimeline);
+			//	(m_epRightLandscapeCompletedHandler.DetachEventHandler(m_tpRightLandscapeTimeline));
+			//	m_tpRightLandscapeTimeline.Clear();
+			//}
+		}
+
+		void OnEntranceAnimationCompleted(
+			DependencyObject pSender,
+			DependencyObject pArgs)
+		{
+			// UNO TODO
+			// m_animationInProgress = false;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5205492)